.gitmodules

```diff
@@ -1,12 +1,15 @@
 [submodule "atmos_cubed_sphere"]
 	path = atmos_cubed_sphere
-	url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
+	#url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
+	url = https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere
```

Comment:
May I ask why you want to point to gfdl dycore in this commit?

Reply:
Dusan's PR 40 is about merging GFDL's dev/emc into EMC's dev/emc. I am doing the same thing as we usually do when we use user branches in PRs - this time the source is GFDL's dev/emc (instead of, for example, climbfuji's my_pr branch) and the target is EMC's dev/emc (as usual). Will be reverted once GFDL_atmos_cubed_sphere 40 is merged.